### DISCUSSION TITLE
Alert Missing

### DISCUSSION THREAD

**Discussion by asmaaelkandoussi** [Security-Onion-Solutions/securityonion#11446] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

20

### Storage for /

300

### Storage for /nsm

automatic

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi all, 

I have installed Security Onion 2.4.10 from iso 
I can't see alerts , How to resolve the problem ? 
I was in SO 2.3.630  without any problem with alert 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
Please follow the Troubleshooting Alerts section of the documentation:
https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts
